docs/docs_style_guide.md

```diff
@@ -0,0 +1,479 @@
+
+#### Block quotes
+
+Use the block quote Markdown for quotes longer than a few sentences.
+
+## R
+
+### References
+
+References should be easy to understand by anyone, not just
+specialists.
+
+They should follow the style guide. When writing a reference:
+
+* don't use italics
+* use single quote marks around titles
+* write out abbreviations in full, for example 'page' not 'p', and
+  'IEEE Transactions on Computers' not 'IEEE Trans. Comp.'
+* use plain English, for example use 'and others' not 'et al'
+* don't use full stops after initials or at the end of the reference
+
+If the reference is available online, make the title a link and
+include the date you accessed the online version.
+
+## S
+
+### Scrum
+
+Upper case when referring to the framework and methon for developing
+products, otherwise use lower case.
+
+### seasons
+
+spring, summer, autumn and winter are lowercase.
+
+### Sentence length
+
+Don't use long sentences. Check and sentences with more than 25 words
+to see if you can split them to make them clearer.
+
+### spaces
+
+Use only one space after a full stop, not 2.
+
+### Speech marks
+
+See 'Quotes and speech marks'.
+
+### steps
+
+See 'Bullets and steps'.
+
+### Summaries
+
+Summaries should:
+
+* be 140 characters or less
+* end with a full stop
+* not repeat the title or body text
+* be clear and specific
+
+## T
+
+### technical terms
+
+Where you need to use technical terms, you can. They're not
+jargon. You just need to explain what they mean the first time you use
+them.
+
+### Temperature
+
+Use Celcius, e.g. 37℃.
+
+### Times
+
+* use 'to' in time ranges, not hyphens, en rules or em dashes. For example, '10am to 11am' not '10–11am'.
+* 5:30pm (not 1730hrs)
+* midnight, not 00:00
+* midday, not 12 noon, noon or 12pm
+* 6 hours 30 minutes
+
+### Titles
+
+Titles should:
+
+* be 65 characters or less
+* be unique, clear and descriptive
+* be front-loaded and optimised for search
+* use a colon to break up longer titles
+* not contain dashes or slashes
+* not have a full stop at the end
+* not use acronyms unless they are well-known, eg EU
+
+## W
+
+### Webpage
+
+One word.
+
+### Wi-Fi
+
+Upper case and hyphenated (trade mark).
+
+### Word document
+
+Upper case, because it's a brand name.
+
+### Words to avoid
+
+Plain English is mandatory for all LiveCode documentation, so please
+avoid using these words:
+
+* deliver (pizzas, post and services are delivered, not abstract
+  concepts like 'improvements' or 'priorities')
+* facilitate (instead, say something specific about how you're
+  helping)
+* focusing
+* impact (don't use this as a synonym for 'have an effect on' or
+  'influence')
+* initiate
+* key (unless it unlocks something. A subject/thing isn't 'key' -
+  it's probably 'important')
+* progress (as a verb - what are you actually doing?)
+* promote (unless you're talking about an ad campaign or some other
+  marketing promotion)
+* robust
+* streamline
+* utilise
+
```

Comment:
Actually (to get all serious about things) "refactor" is an umbrella for so many different things that maybe it really should be on the list unless it's in a very explicit context.